lib/utils/matrix_sdk_extensions/matrix_locals.dart

```diff
@@ -311,4 +311,10 @@ class MatrixLocals extends MatrixLocalizations {
 
   @override
   String get unknownUser => l10n.user;
+
+  @override
+  String hasKnocked(String targetName) {
+    // TODO: implement hasKnocked
+    throw UnimplementedError();
+  }
```

Comment:
why don't you use it?

Reply:
it's new method from upstream sdk, currently no need to focus on this